### PR TITLE
Updated link handler for toolwindow to support hyperlinks and mailto

### PR DESCRIPTION
**Pull request by emessiha**
If statment and corresponding logic was missing to handle normal hyperlinks. Also fixed a typo in one of the text messages to user

### CODE REVIEW

File: src/main/java/com/reshiftsecurity/plugins/intellij/actions/HelpAction.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ private static StringBuilder createHelpInfo() {
 		ret.append(String.format("Support & Feedback: <a href='mailto:%s'>dev@reshiftsecurity.com</a>", PluginConstants.RESHIFT_DEV_EMAIL));
 		ret.append("<br/>");
 		ret.append("<br/>");
-		ret.append("Would you like to integrate Reshift into your CI pipline? Sign up today for free on <a href='")
+		ret.append("Would you like to integrate Reshift into your CI pipeline? Sign up today for free on <a href='")
```
Comment (ohamel-softwaresecure) [inline src/main/java/com/reshiftsecurity/plugins/intellij/actions/HelpAction.java:123]: is there a tool to spell check our string literals?

**Review by smiucin-reshift — APPROVED** [review]

File: src/main/java/com/reshiftsecurity/plugins/intellij/gui/toolwindow/view/ToolWindowPanel.java [hunk 2]
```diff
@@ -478,6 +482,24 @@ public void hyperlinkUpdate(@NotNull final Notification notification, @NotNull f
...
+				} else {
+					try {
+						URI eventURI = e.getURL().toURI();
+						if (eventURI.getScheme().equalsIgnoreCase("mailto")) {
+							Desktop desktop = Desktop.getDesktop();
+							if (desktop.isSupported(Desktop.Action.MAIL)) {
+								try {
+									desktop.mail(eventURI);
+								} catch (IOException emailEx) {
+									emailEx.printStackTrace();
```
Comment (ohamel-softwaresecure) [inline src/main/java/com/reshiftsecurity/plugins/intellij/gui/toolwindow/view/ToolWindowPanel.java:494]: printStackTrace in production?

File: src/main/java/com/reshiftsecurity/plugins/intellij/gui/toolwindow/view/ToolWindowPanel.java [hunk 3]
```diff
@@ -478,6 +482,24 @@ public void hyperlinkUpdate(@NotNull final Notification notification, @NotNull f
...
+								}
+							}
+						} else {
+							BrowserUtil.browse(e.getURL());
+						}
+					} catch (URISyntaxException uriSyntaxException) {
```
Comment (ohamel-softwaresecure) [inline src/main/java/com/reshiftsecurity/plugins/intellij/gui/toolwindow/view/ToolWindowPanel.java:500]: same

File: src/main/java/com/reshiftsecurity/plugins/intellij/actions/HelpAction.java (hunk 1 above)
Comment (emessiha) [reply src/main/java/com/reshiftsecurity/plugins/intellij/actions/HelpAction.java:123]: Yes, will look into integrating something in our CD pipeline

**Pull request merged**